x-pack/legacy/plugins/alerting/server/routes/create.ts

```diff
@@ -29,6 +30,7 @@ export function createAlertRoute(server: Hapi.Server) {
         },
         payload: Joi.object()
           .keys({
+            enabled: Joi.boolean().default(true),
```

Comment:
Wondering if the default should be `false`

Reply:
We could also remove the default and make it explicit. I'm not sure what behaviour is best? Currently they're enabled and get scheduled right away.

Reply:
Would be surprising to me if I create an alert without an explicit `enabled` set, and it made it disabled by default.  Should either be required, or default to enabled, seems like.

Reply:
As discussed with @bmcconaghy, we'll leave the behaviour to the same that is in master (enabled as soon as you create an alert). And if ever we hear different expectations, we'll do the change in a follow up PR.